models/category/src/category-draft/presets/sample-data-goodstore/home-accents.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  LocalizedString,
+  KeyReference,
+} from '@commercetools-test-data/commons';
+import type { TCategoryDraftBuilder, TCategoryDraft } from '../../../types';
+import empty from '../empty';
+import roomDecor from './room-decor';
+
+const roomDecorDraft = roomDecor().build<TCategoryDraft>();
+
+const homeAccents = (): TCategoryDraftBuilder =>
+  empty()
+    .name(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('Home Accents')
+        ['en-US']('Home Accents')
+        ['de-DE']('Home Akzente')
+    )
+    .slug(
+      LocalizedString.presets
+        .empty()
+        ['en-GB']('home-accents')
+        ['en-US']('home-accents')
+        ['de-DE']('home-akzente')
+    )
+    .key('home-accents')
+    .orderHint('.5')
+    .parent(KeyReference.presets.category().key(roomDecorDraft.key!));
```

Comment:
Should the parent be `furniture.ts`?

Reply:
https://mc.europe-west1.gcp.commercetools.com/thegoodstore/categories/caf67eff-7fdd-4993-8c6a-7c08be501cd6/general

Reply:
Yep yep, my mistake - thanks!